packages/puppeteer-core/src/cdp/AriaQueryHandler.ts

```diff
@@ -35,6 +35,10 @@ const isKnownAttribute = (
 const ATTRIBUTE_REGEXP =
   /\[\s*(?<attribute>\w+)\s*=\s*(?<quote>"|')(?<value>\\.|.*?(?=\k<quote>))\k<quote>\s*\]/g;
 const parseARIASelector = (selector: string): ARIASelector => {
+  if (selector.length > 1_000) {
```

Comment:
could we go with 1k here?

Reply:
s/1k/10k

Reply:
Let me try, though I think 1K is reasonable as well.